Win10.ps1

```diff
@@ -1975,6 +1976,18 @@ Function InstallMediaPlayer {
 	Enable-WindowsOptionalFeature -Online -FeatureName "WindowsMediaPlayer" -NoRestart -WarningAction SilentlyContinue | Out-Null
 }
 
+# Uninstall Internet Explorer
+Function UninstallInternetExplorer {
+	Write-Output "Uninstalling Internet Explorer..."
+	Disable-WindowsOptionalFeature -Online -FeatureName "Internet-Explorer-Optional-amd64" -NoRestart -WarningAction SilentlyContinue | Out-Null

```

Comment:
I'm not sure what the name would be for a 32bit system, but all the following guides mention only this name.

- https://gallery.technet.microsoft.com/scriptcenter/Script-to-remove-Internet-e8327b13
- https://www.tenforums.com/tutorials/27095-install-uninstall-internet-explorer-windows-10-a.html
- https://4sysops.com/archives/uninstall-internet-explorer-on-windows-10/

Reply:
Yup, `Internet-Explorer-Optional-x86`. I had to download 32bit from MSDN and to be honest, this was the first time I have actually seen them. I think I don't want to support them anyway :)